packages/teleport-test/src/standalone.ts

```diff
@@ -91,6 +93,19 @@ const run = async () => {
           ...packerOptions.publishOptions,
           projectSlug: 'teleport-project-next-cms',
         },
+        plugins: [
+          new ProjectPluginNextCache({
+            routeMappers: {
+              /* tslint:disable no-invalid-template-strings */
+              bogpost: ['/bogpost/${id}', '/bogpost'],
```

Comment:
I assume this is for testing only, and in production we'll get this data from GUI, right?

Reply:
yes 👍